tembo-stacks/src/stacks/specs/analytics.yaml

```diff
@@ -23,21 +22,45 @@ postgres_config:
   - name: pg_stat_statements.track
     value: top
   - name: shared_preload_libraries
-    value: pg_stat_statements,pg_cron
+    value: pg_stat_statements, pg_cron, pg_later, pg_parquet
   - name: track_io_timing
     value: 'on'
   - name: wal_level
     value: logical
+  - name: pglater.host
+    value: postgresql:///postgres?host=/controller/run
 trunk_installs:
+  - name: hydra_columnar
+    version: 1.1.1
```

Comment:
fyi - this will create another blocker for adding pg17 support to this stack. pg17 support does not appear to be a priority for that project https://github.com/hydradatabase/hydra/issues/272

Reply:
Yeah, I'm aware. Do we have the ability to do version specific extensions? i.e. the definition of this stack for pg16 and pg17 is different. I think no, just wanted to ask.

Reply:
No, we do not.